ui/apps/platform/src/Components/PatternFly/BacklogListSelector.tsx

```diff
@@ -7,10 +7,13 @@ import {
     EmptyStateVariant,
     Flex,
     FormGroup,
+    EmptyStateHeader,
+    EmptyStateFooter,
+    Icon,
 } from '@patternfly/react-core';
 import { CubesIcon, MinusCircleIcon, PlusCircleIcon } from '@patternfly/react-icons';
 
-import { BaseCellProps, TableComposable, Tbody, Td, Tr } from '@patternfly/react-table';
+import { BaseCellProps, Table /* data-codemods */, Tbody, Td, Tr } from '@patternfly/react-table';
```

Comment:
Do we need to leave the code-mod comment in here, or is this a sign we need to do something more for the migration?
```suggestion
import { BaseCellProps, Table, Tbody, Td, Tr } from '@patternfly/react-table';
```

Reply:
Fixed globally in #10563